bin/verify-exercises

```diff
@@ -20,16 +20,20 @@
 
 # Verify the Concept Exercises
 for concept_exercise_dir in ./exercises/concept/*/; do
-    if [ -d $concept_exercise_dir ]; then
-        echo "Checking $(basename "${concept_exercise_dir}") exercise..."
-        # TODO: run command to verify that the exemplar solution passes the tests
+    if [ -d "$concept_exercise_dir" ]; then
+        slug=$(basename "${practice_exercise_dir}")
+        echo "Checking ${slug} exercise..."
+        cp "${concept_exercise_dir}"/.meta/exemplar.sql "${concept_exercise_dir}/${slug}".sql
+        bin/verify-exercise-example "${slug}" "${concept_exercise_dir}" "${concept_exercise_dir}"
```

Comment:
Should the verify script copy the example/examplar file? That seems like it should be part of the verify logic.

Should something restore the `${slug}.sql` file?

Reply:
In my understanding, the container is ditched after the testing, so there is no need to preserve the data.

Reply:
Maintainers may want to verify things locally, too!

Reply:
That's true. I never used the lcoal setup in the c++ track, because it does not fit my work flow. But here we could preserve it.